src/careamics/transforms/n2v_manipulate.py

```diff
@@ -142,5 +143,8 @@ def __call__(
         else:
             raise ValueError(f"Unknown masking strategy ({self.strategy}).")
 
+        # TODO: Output does not match other transforms, how to resolve?
+        #     - Don't include in Compose and apply after if algorithm is N2V?
+        #     - or just don't return patch? but then mask is in the target position
```

Comment:
The patch needs to be returned, what I always wondered was whether we could return `patch * mask` rather than `patch, mask`.

However, there was a reason in the loss. So we might want to have a closer look again and see if we can reimplement the loss in a way where we don't need the both patch and mask individually.